parca.yaml

```diff
@@ -10,7 +10,12 @@ debug_info:
 
 scrape_configs:
   - job_name: "default"
-    scrape_interval: "3s"
-    scrape_timeout: "2s"
+    scrape_interval: "60s"
+    scrape_timeout: "45s"
     static_configs:
       - targets: [ '127.0.0.1:7070' ]
+    profiling_config:
+      pprof_config:
+        fgprof:
+          enabled: true
+          path: /debug/pprof/fgprof
```

Comment:
I don't think we should be changing the default config here. 
Most applications won't have this extra profile type by default and thus it would add more confusion. 
Instead I would propose to add this to our documentation with this example snippet.

Reply:
makes sense to me 👍

Reply:
I'll remove these config changes from the PR, and add a suggestion to the `parca-dev/parca.dev` docs